src/components/text-input/index.md.njk

```diff
@@ -188,3 +188,7 @@ For example, ‘How much you earn a week must include pence, like 123.45 or 156.
 
 Say ‘[whatever it is] must not include pence, like 123 or 156’.<br>
 For example, ‘How much you earn a week must not include pence, like 123 or 156’.
+
+## Research on this component
+
+Read a blog post about [why we do not recommend using input type="number"](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/).
```

Comment:
Would it better to say "the problems we discovered with `<input type="number">`" or something similar? I don't think we're going to go as far as saying we don't recommend it, I think teams have to make that decision themselves based on their use case.

Reply:
Yes, I like that - have updated the link text